CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing to BETER Feed Testing Sandbox (BFTS)
+
+## Introduction
+
+Thank you for considering contributing to the BETER Feed Testing Sandbox (BFTS) project! This document provides guidelines and instructions to help you contribute effectively.
+
+## Table of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+2. [How to Contribute](#how-to-contribute)
+   - [Fork the Repository](#fork-the-repository)
+   - [Create a Branch](#create-a-branch)
+   - [Make Changes](#make-changes)
+   - [Test Your Changes](#test-your-changes)
+   - [Submit a Pull Request](#submit-a-pull-request)
+3. [Guidelines for Development](#guidelines-for-development)
+   - [Resource Requirements](#resource-requirements)
+   - [Load testing resources requirements](#load-testing-resources-requirements)
+   - [Versioning Process](#versioning-process)
+4. [Reporting Issues](#reporting-issues)
+5. [Getting Help](#getting-help)
+
+## Code of Conduct
+
+This project adheres to a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [email@example.com](mailto:email@example.com).
+
+## How to Contribute
+
+### Fork the Repository
```

Comment:
It's a copy-paste from the section https://github.com/BETER-CO/beter-feed-testing-sandbox/blob/main/doc/RUNNING_APPS.md#local-development

Please, add the link to this section to explain how to launch services for development. Everything else isn't mandatory.